docs/protocol-drafts/BECKN-002-Payments-On-Beckn-Enabled-Networks.md

```diff
@@ -155,7 +155,7 @@ Immediate payment on order placement to a UPI Endpoint
 ```
 {
     "uri": "payto://upi/example@upi?amount=$currency:$value&message=hello",
-    "method": "PAYTO",
+    "tl_method": "UPI",
```

Comment:
this is incorrect. Method will still be PAYTO. Maybe we can add one more example using a UPI url? `upi://....`

Reply:
Changed tl_method to PAYTO and added example for UPI